frontend/src/metabase/query_builder/components/template_tags/TagEditorParam.tsx

```diff
@@ -324,9 +366,7 @@ export class TagEditorParam extends Component {
                   ? parameter || {
                       fields: [],
                       ...tag,
-                      type:
-                        tag["widget-type"] ||
-                        (tag.type === "date" ? "date/single" : null),
+                      type: tag["widget-type"] || null,
```

Comment:
TypeScript pointed out that this expression is redundant:

```js
tag.type === "date" ? "date/single" : null
```

We're doing a `tag.type === "text" || tag.type === "dimension"` above and there's no way `tag.type` can be `"date"`